src/views/Calendar.vue

```diff
@@ -28,13 +33,22 @@ export default {
   name: 'Calendar',
   data() {
     return {
-      weeklyOrMonthlyShow: 'weekly',
+      calendarShowOptions: [
```

Comment:
다음과 같이 작성하면 변수명을 좀 더 표현력 있게 가져가면서도, 문자열 비교 `(v-show="calendarShow=='weekly'" -> v-show="selectedType === calendarTypes.WEEKLY")` 를 피할 수 있습니다.

```suggestion
      calendarTypes: {
        WEEKLY: {value: 'weekly', label: '주'},
        MONTHLY: {value: 'monthly', label: '월'},
      },
      selectedType: calendarTypes.WEEKLY,
```